### ISSUE TITLE
Minor: the artist previously known as balkan

### ISSUE THREAD

**Issue by aral** [sonnyp/Commit#35]
Hey Sonny,

Congrats on the work you’re doing on Commit and on supporting it in GNOME circles. Sorry I was slow to reply to your messages; had other things on – happy you ran with it as I’m focussing any limited time I have left over from the Small Web work on elementary OS-specific stuff. So it’s nice to see this available for the wider community.

One request: could you capitalise my last name in the README, etc.? (Aral Balkan, not Aral balkan.) Not a biggie, I just don’t want people to think I use some weird stylised way of writing it :)

Looking forward to seeing where you take it and keep your eyes peeled for something specific to elementary OS in the near future from me ;)

Take care + 💕

**Comment by sonnyp** [comment]
:wave: 

For clarity and because I don't want readers to think I forked for illegitimate reasons; my messages were pull requests and you didn't reply at all.

It's not a problem and I don't feel like you owe me anything, on the contrary, I'm grateful for Gnomit :heart: 

I took extra care in making sure to credit you properly. I used the spelling from https://github.com/small-tech/gnomit#copyright

Fixed in https://github.com/sonnyp/Commit/commit/7d0530984b18018f5f58fb48a42887814d0b3c62

Cheers!
